Sources/Honeycomb/Honeycomb.swift

```diff
@@ -168,5 +168,9 @@ public class Honeycomb {
         if #available(iOS 13.0, macOS 12.0, *) {
             MXMetricManager.shared.add(self.metricKitSubscriber)
         }
+
+        if options.debug {
```

Comment:
Would it make more sense to put this at the top of the function, in case the options cause some kind of error, or if we later add some other debug lines during configuration?